src/types/webcompat-settings.d.ts

```diff
@@ -29,7 +29,30 @@ export interface WebCompatSettings {
       undefined: string[];
     };
   };
+  modifyLocalStorage?: {
+    state: State;
+    changes: {}[];
```

Comment:
Ah I see, give this like an explicit object shape perhaps?

Reply:
Yep, that does it.